src/Integration.Vsix/Suppression/SuppressionManager.cs

```diff
@@ -68,9 +69,9 @@ private void RefreshSuppresionHandling()
                     CleanupSuppressionHandling();
                 }
             }
-            catch(Exception)
+            catch(Exception ex)
             {
-                // TODO: log exceptions to the output window
+                this.sonarLintOutput.Write($"Failed to refresh suppression handling: {ex.Message}");
```

Comment:
Implemented the TODO, since the suppression manager already has access to a _ISonarLintOutput_ instance.